src/ceph-volume-zfs/bin/ceph-volume-zfs

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+from ceph_volume_zfs import main
+if __name__ == '__main__' and __package__ is None:
+    from os import sys, path
+    sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+    # print(sys.path)
```

Comment:
can you remove this?

Reply:
@alfredodeza 
This is temporary debugging for me.
I will clean up all thes commented prints in a final sweep, once you think it is good to go.